src/redux/reducer.js

```diff
@@ -0,0 +1,39 @@
+import { ADD_PATTERN, DELETE_PATTERN, MODIFY_PATTERN } from "./actions";
+
+const initialState = {
+  patterns: [
+
+    // Dummy Pattern
+    {
+      key: "cbdjvcdscnds",
+      name: "Test Pattern",
+      example: "HYI-853",
+      regex: "[A-Z]{3}-[0-9]{3}",
```

Comment:
Yes, It's a typo, I had written 5 ( value of the length of the second entry in regexArray ) instead of 3 by mistake. I have corrected it in my latest commit.